### DISCUSSION TITLE
Adding Images

### DISCUSSION THREAD

**Discussion by pierceglenn12345** [rabbibotton/clog#331] [General]
I have added a png image(image.png) to the path with the other clog images (~/quicklisp/local-projects/dashboard/www/img/image.png, clogicon.png is also here). When I go to top.clog and select the element (app-image) that holds the image and change "image-url" from http://127.0.0.1:8080/img/clogicon.png to http://127.0.0.1:8080/img/image.png, all I get is the alternative text "Add image url". If I revert and change image to clogicon in the url above from image, I get the original clog image rendered correctly. What could I be doing wrong?

**Comment by rabbibotton** [comment]
Sorry just saw, best and easiest use the CLOG Builder image to html data and just pop the results of that into your URL and you image is now part of you source. Second should change the url to "/img/image.png"
